devicon.json

```
@@ -11278,12 +11278,26 @@
                 "plain-wordmark"
             ]
         },
-        "color": "#f7a41d",
-        "aliases": [
-            {
-                "base": "original",
-                "alias": "plain"
-            }
-        ]
+        "color": "#294172", 
+        "aliases": [] 
+    },
+    {
+        "name": "stackblitz",
```

Add missing altnames
```suggestion
        "name": "stackblitz",
        "altnames": [],
```